src/campsite-info/campsite-info.component.tsx

```diff
@@ -1,6 +1,12 @@
 import React, { FunctionComponent } from "react";
 import { CampHeader } from "./components/camp-header";
+import { CampDescription } from "./components/camp-description";
 
 export const CampsiteInfo: FunctionComponent = () => {
-    return <CampHeader region="region" park="park" zone="zone" site="site" logout={() => {}}/>;
+  return (
+    <div>
+      <CampHeader region="region" park="park" zone="zone" site="site" logout={() => {}}/>
+      <CampDescription description="description about campsite"/>
```

Comment:
The description field might not need to be its own component, depending on how complex it might turn out to be.